test/pkg.jl

```diff
@@ -100,6 +100,8 @@ temp_pkg_dir() do
     str = chomp(takebuf_string(iob))
     @test endswith(str, string(Pkg.installed("Example")))
     @test isempty(Pkg.dependents("Example"))
+    # Checking to see dependency reading
+    @test Pkg.dependencies("Example") == Dict("julia" => Pkg.Types.VersionSet(v"0.3.0", typemax(VersionNumber)))
```

Comment:
It's not. But I think the package manager should report on all of the packages it has knowledge of, which (I think) should include packages the user cloned manually.

I agree that there should be different accessors for getting this data from metadata or the installed package repositories themselves. I just don't think it's wrong for the main API of the package manager to provide something a little more curated by default. I could additionally update to dependents to allow for different kinds of queries as well? Maybe an `:installed, :available, :all` switch? Or user installed packages requires files could be available in metadata?